src/NewRelicBackgroundJobMiddleware.php

```diff
@@ -20,6 +20,6 @@ public function handle(Request $request, Closure $next)
         // Mark the request as a background job
         $this->newRelic->backgroundJob();
 
-        return parent::handle($request, $next);
+        return $next($request);
```

Comment:
I have to check how we use this middleware, I'm not 100% this is correct since it changes the logic.

Reply:
Hm, i see. I think i didn't face that dilema because i am using version 1 yet since my lumen it's a little bit old. What do you suggest?

Reply:
I would drop this change for now

Reply:
Ok, no problem, thanks.